.chloggen/remove-promtail-receiver.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
```

Comment:
Given that it was not part of any releases so far, it would be more appropriate to just remove the changelog entry we have for it